notebooks/contributivity_method_comparison.ipynb

```diff
@@ -0,0 +1,1142 @@
+{
```

Comment:
I am getting an error on this one, is this a path error from `root_folder = exp_path = Path().absolute() / "experiments" ` or that I have to run something else before?  
It is looking for the folder "experiments" inside "notebooks" folder: `'/home/zen/dev/distributed-learning-contributivity/notebooks/experiments'`

Reply:
Same for the others notebooks, I might have missed a step, please let me know if so

Reply:
I did no commit the results file so it will not work

Reply:
These notebooks are "base" notebooks that can be used to analyse the results but they are not a portfolio like we discussed in https://github.com/SubstraFoundation/distributed-learning-contributivity/issues/146

Reply:
Sorry, my bad, thank for the clarification!